src/pages/VenuePage/VenuePage.tsx

```diff
@@ -173,20 +263,44 @@ export const VenuePage: React.FC = () => {
 
   // useVenueAccess(venue, handleAccessDenied);
 
-  if (venueRequestStatus && !venue) {
+  if (isVenueLoaded && !venue) {
     return <>This venue does not exist</>;
   }
 
-  if (!venue || !venueId) {
-    return <LoadingPage />;
+  if (!venueId) {
+    // this should be happening only if invalid url param
+    // no sense in displaying loading page, another message or action is needed
+    // @debt use/display proper message/action when !venueId
+    return null;
+  }
+
+  if (!venue) {
+    return (
+      <Suspense fallback={<></>}>
+        <LoadingPage />
+      </Suspense>
+    );
   }
```

Comment:
Lazy loading `LoadingPage` and wrapping it with `Suspense` here doesn't make any sense.

As per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
No longer lazy loaded, removed `Suspense` wrapper f57638d5e088c8412a2d7eaf65a3a30942f29641